Source/WebCore/dom/Document.cpp

```diff
@@ -8602,6 +8602,10 @@ void Document::setHasSkippedResizeObservations(bool skipped)
 
 void Document::updateResizeObservations(Page& page)
 {
+    if (quirks().shouldSilenceResizeObservers()) {
+        addConsoleMessage(MessageSource::Other, MessageLevel::Info, "ResizeObservers silenced due to: rdar://110097836"_s);
```

Comment:
This *should* only generate a message when the background occurs, so I don't think this will create too much logging.

Reply:
If we're going to log the console, we shouldn't log a radar number that is opaque to non-Apple folks. File a bugzilla, and reference that (if anything).

Reply:
Agreed with Simon, we can create a public bug for it. 
Thanks @sgill26 for the log message ❤️ 
Do you want me to create the public bug?

Reply:
see https://github.com/WebKit/WebKit/pull/15346#pullrequestreview-1502110638

Reply:
Changed